sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AzureOauth2RefreshTokenGrantRequestEntityConverter.java

```diff
@@ -19,10 +20,12 @@
     @SuppressWarnings("unchecked")
     public RequestEntity<?> convert(OAuth2RefreshTokenGrantRequest refreshTokenGrantRequest) {
         RequestEntity<?> result = super.convert(refreshTokenGrantRequest);
-        Optional.ofNullable(result)
-            .map(HttpEntity::getBody)
-            .map(b -> (MultiValueMap<String, String>) b)
-            .ifPresent(body -> body.add("scope", scopeValue(refreshTokenGrantRequest)));
+        if (!AADClientRegistrationRepository.isDefaultClient(refreshTokenGrantRequest.getClientRegistration())) {
+            Optional.ofNullable(result)
+                .map(HttpEntity::getBody)
+                .map(b -> (MultiValueMap<String, String>) b)
+                .ifPresent(body -> body.add("scope", scopeValue(refreshTokenGrantRequest)));
+        }
```

Comment:
After https://github.com/Azure/azure-sdk-for-java/pull/18297 merged, `AzureOauth2RefreshTokenGrantRequestEntityConverter.java` can be deleted.